mfr/extensions/pdb/render.py

```diff
@@ -16,9 +19,14 @@ class PdbRenderer(extension.BaseRenderer):
         ]).get_template('viewer.mako')
 
     def render(self):
+        self_url = urlparse(self.metadata.download_url)
```

Comment:
The code here and in the `render` methods for the pdf and video exporters are almost identical.  Could you move this into a decorator/utility method/base class/whatever to avoid the code duplication?

Reply:
Completed.